test/name-test.js

```diff
@@ -63,39 +62,6 @@ describe('tmp', function () {
           });
         });
       });
-      describe('on issue #246', function () {
```

Comment:
Same here, tests must stay in place, yet test for the presence of the quotes.

Reply:
Please see [a831713](https://github.com/raszi/node-tmp/pull/278/commits/a831713743a1e370ebcda62f0709fdda9c0dd9d6).